roles/mongo_7_0/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
     update_cache: true
 
 - name: Set break_system_packages if Ubuntu version >= 24
-  set_fact:
+  ansible.builtin.set_fact:
     PIP_EXTRA_ARGS: "{{ PIP_EXTRA_ARGS }} --break-system-packages"
   when: ansible_distribution == 'Ubuntu' and ansible_distribution_version is version('24', '>=')
```

Comment:
After some tough, if we are going to have a task for this we can probably remove the check altogether in the machine:

```yaml
- name: Ignore PEP 668
  ansible.builtin.file:
    path: /usr/lib/python3.12/EXTERNALLY-MANAGED
    state: absent
```